tests/register.spec.coffee

```diff
@@ -1,9 +1,6 @@
-requestMock = require 'requestmock'
-mockery = require 'mockery'
-
-requestMock.log(false)
-mockery.enable(warnOnUnregistered: false)
-mockery.registerMock('request', requestMock)
+Promise = require 'bluebird'
+global.Promise = Promise
```

Comment:
Without this, fetch returns a Node promise, not a bluebird promise. I've done it this way here primarily because it's the same approach we've used in resin-request.

If we directly drop this, we have to do one of:

1. Wrap every `fetch()`d promise with `Promise.resolve` (with `Promise = require('bluebird')`)
2. Set `fetch.Promise` instead (which would still set the Promise used for fetch globally, but not Promise for the whole Node process)
3. Stop using the bluebird API on `fetch()`d promises.